http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=476571
--- neon27-0.28.2.orig/src/ne_uri.c
+++ neon27-0.28.2/src/ne_uri.c
@@ -409,7 +409,7 @@
             } else {
                 if (relative->path[0] == '/') {
                     target->path = remove_dot_segments(relative->path);
-                } else {
+                } else if (base->path) {
                     char *merged = merge_paths(base, relative->path);
                     target->path = remove_dot_segments(merged);
                     ne_free(merged);
