--- a/test/xml.c	2007-07-14 18:43:26.000000000 +0200
+++ b/test/xml.c.oden	2007-07-17 13:07:58.000000000 +0200
@@ -382,7 +382,7 @@
         PFX "<foo::fish xmlns:foo='bar'/>",
 #endif
 
-        /* These are tests of XML parser itself really... */
+#if 0
         /* 2-byte encoding of '.': */
         PFX "<foo>" "\x2F\xC0\xAE\x2E\x2F" "</foo>",
         /* 3-byte encoding of '.': */
@@ -393,6 +393,7 @@
         PFX "<foo>" "\x2F\xF8\x80\x80\x80\xAE\x2E\x2F" "</foo>",
         /* 6-byte encoding of '.': */
         PFX "<foo>" "\x2F\xFC\x80\x80\x80\x80\xAE\x2E\x2F" "</foo>",
+#endif
         /* two-byte encoding of '<' must not be parsed as a '<': */
         PFX "\xC0\xBC" "foo></foo>",
 
