diff -up neon-0.34.0/neon-config.in.4~ neon-0.34.0/neon-config.in
--- neon-0.34.0/neon-config.in.4~	2024-11-23 11:53:08.000000000 +0100
+++ neon-0.34.0/neon-config.in	2024-11-24 15:23:32.723435812 +0100
@@ -5,7 +5,11 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
 includedir=@includedir@
-libdir=@libdir@
+
+libdir=$(pkg-config --variable=libdir neon)
+CFLAGS=$(pkg-config --cflags neon)
+LIBS=$(pkg-config --libs neon)
+has_lfs=$(pkg-config --variable=has_lfs neon)
 
 usage()
 {
@@ -16,14 +20,13 @@ Known values for OPTION are:
 
   --prefix=DIR		change neon prefix [default $prefix]
   --libs		print library linking information
-  --la-file		print location of libtool .la file
   --cflags		print pre-processor and compiler flags
   --help		display this help and exit
   --version		output version information
   --support FEATURE     exit with success if feature is supported
 
  Known features: 
-    dav [@NE_FLAG_DAV@], ssl [@NE_FLAG_SSL@], zlib [@NE_FLAG_ZLIB@], ipv6 [@NE_FLAG_IPV6@], lfs [@NE_FLAG_LFS@],
+    dav [@NE_FLAG_DAV@], ssl [@NE_FLAG_SSL@], zlib [@NE_FLAG_ZLIB@], ipv6 [@NE_FLAG_IPV6@], lfs [${has_lfs}],
     i18n [@NE_FLAG_I18N@], ts_ssl [@NE_FLAG_TS_SSL@], gssapi [@NE_FLAG_GSSAPI@], libpxy [@NE_FLAG_LIBPXY@]
 
 EOF
@@ -69,20 +72,15 @@ while test $# -gt 0; do
 	;;
 
     --cflags)
-       	echo -I${includedir}/neon @NEON_CFLAGS@
+       	echo ${CFLAGS}
        	;;
 
     --libs)
-        LIBS="-lneon @NEON_LIBS@"
 	# Don't add standard library paths
         if test "$prefix" != "/usr"; then
            LIBS="-L${libdir} ${LIBS}"
         fi
-	echo @user_LDFLAGS@ ${LIBS}
-	;;
-
-    --la-file)
-	echo ${libdir}/libneon.la
+	echo ${LIBS}
 	;;
 
     --support)
@@ -95,7 +93,7 @@ while test $# -gt 0; do
 	zlib|ZLIB) support @NE_FLAG_ZLIB@ ;;
 	ipv6|IPV6) support @NE_FLAG_IPV6@ ;;
 	dav|DAV) support @NE_FLAG_DAV@ ;;
-	lfs|LFS) support @NE_FLAG_LFS@ ;;
+	lfs|LFS) support ${has_lfs} ;;
 	i18n|I18N) support @NE_FLAG_I18N@ ;;
 	ts_ssl|TS_SSL) support @NE_FLAG_TS_SSL@ ;;
 	*) support no ;;
diff -up neon-0.34.0/neon.pc.in.4~ neon-0.34.0/neon.pc.in
--- neon-0.34.0/neon.pc.in.4~	2024-11-23 11:53:08.000000000 +0100
+++ neon-0.34.0/neon.pc.in	2024-11-24 15:22:50.452983729 +0100
@@ -2,6 +2,7 @@ prefix=@prefix@
 exec_prefix=@exec_prefix@
 libdir=@libdir@
 includedir=@includedir@
+has_lfs=@NE_FLAG_LFS@
 
 Name: neon
 Description: HTTP/WebDAV client library
